samples/AppConfig/AppConfigFramework/ConfigInfo.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Web;
+using System.Web.SessionState;
+using Microsoft.AspNetCore.SystemWebAdapters;
+
+namespace AppConfigFramework;
+
+public class ConfigInfo : IHttpHandler, IRequiresSessionState
+{
+    public bool IsReusable => true;
+
+    public void ProcessRequest(HttpContext context)
+    {
+        context.Response.ContentType = "application/json";
+        context.Response.StatusCode = 200;
+
+        context.Response.Write(JsonSerializer.Serialize(new
+        {
+            Setting1 = AppConfiguration.GetSetting("Setting1"),
```

Comment:
Is the plan for this to look the same on Core? Or is the plan to have folks use DI Configuration there?